packages/portal/spec-renderer-mini/src/components/SpecRendererMini.vue

```diff
@@ -158,7 +158,13 @@ const untaggedItems = computed((): SpecItemType[] => {
 })
 
 const getSectionDescription = (section: string): string => {
-  return props.tags.filter((item: SpecTag) => item.name === section)?.[0].description || ''
+  const tagData = props.tags.filter((item: SpecTag) => item.name === section)
```

Comment:
nit: `.find` might be efficient here

Reply:
How did I forget about this function?? Thank you!